packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts

```diff
@@ -50,8 +37,7 @@ describe('LWC Hovers', () => {
         'server.js'
       )
     );
-    client.start();
```

Comment:
https://github.com/Microsoft/vscode-languageserver-node?tab=readme-ov-file#3170-protocol-800-json-rpc-800-client-and-800-server